code/modules/mob/living/carbon/xenomorph/Powers.dm

```diff
@@ -998,7 +998,7 @@
 		handle_ventcrawl(pipe)
 
 
-/mob/living/carbon/Xenomorph/proc/xeno_transfer_plasma(atom/A, amount = 50, transfer_delay = 20, max_range = 2)
+/mob/living/carbon/Xenomorph/proc/xeno_transfer_plasma(atom/A, amount, transfer_delay = 20, max_range = 2)
```

Comment:
Since you removed the default, are you sure all places calling it are using the parameters correctly?